dex/models/_projects/paraswap/arbitrum/paraswap_v5_arbitrum_trades.sql

```diff
@@ -7,10 +7,10 @@
     file_format = 'delta',
     incremental_strategy = 'merge',
     unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
-    post_hook='{{ expose_spells(\'["arbitrum"]\',
-                                "project",
-                                "paraswap_v5",
-                                \'["springzh"]\') }}'
+    post_hook='{{ expose_spells(blockchains = \'["arbitrum"]\',
+                                spell_type = "project",
+                                spell_name = "paraswap_v5",
+                                contributors = \'["springzhang"]\') }}'
```

Comment:
Hi @jeff-dude,
should I remove the `post_hook` in blockchain-version-level spells?